app/src/main/java/org/mozilla/focus/utils/StatusBarUtils.java

```diff
@@ -0,0 +1,31 @@
+package org.mozilla.focus.utils;
```

Comment:
Please add our license header at the top of the file:

```
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Reply:
Done